src/rest/APIRequest.js

```diff
@@ -41,17 +42,20 @@ class APIRequest {
       for (const file of this.options.files) if (file && file.file) body.append(file.name, file.file, file.name);
       if (typeof this.options.data !== 'undefined') body.append('payload_json', JSON.stringify(this.options.data));
       if (!browser) headers = Object.assign(headers, body.getHeaders());
-    } else if (this.options.data != null) { // eslint-disable-line eqeqeq
+    } else if (this.options.data != null) { // eslint-disable-line 
```

Comment:
Why was this changed?

Reply:
I thought it was a typo or something, I can just readd it back if I'm wrong

Reply:
It tells eslint to ignore that the next line is not using strict equality/triple-equals (`eqeqeq`)

Not a typo

Reply:
Thanks for clarification, will update this